apps/condo/migrations/20211118192321-0082_auto_20211118_1423.js

```diff
@@ -0,0 +1,53 @@
+// auto generated by kmigrator
+// KMIGRATOR:0082_auto_20211118_1423:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMi41IG9uIDIwMjEtMTEtMTggMTQ6MjMKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAwODFfYXV0b18yMDIxMTExOF8xMTU1JyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSd0aWNrZXQnLAogICAgICAgICAgICBuYW1lPSdvcmRlcicsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5JbnRlZ2VyRmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3RpY2tldGNoYW5nZScsCiAgICAgICAgICAgIG5hbWU9J29yZGVyRnJvbScsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5JbnRlZ2VyRmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3RpY2tldGNoYW5nZScsCiAgICAgICAgICAgIG5hbWU9J29yZGVyVG8nLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuSW50ZWdlckZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSd0aWNrZXRoaXN0b3J5cmVjb3JkJywKICAgICAgICAgICAgbmFtZT0nb3JkZXInLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuSW50ZWdlckZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgIF0K
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field order to ticket
+--
+ALTER TABLE "Ticket" ADD COLUMN "order" integer NULL;
+--
+-- Add field orderFrom to ticketchange
+--
+ALTER TABLE "TicketChange" ADD COLUMN "orderFrom" integer NULL;
+--
+-- Add field orderTo to ticketchange
+--
+ALTER TABLE "TicketChange" ADD COLUMN "orderTo" integer NULL;
+--
+-- Add field order to tickethistoryrecord
+--
+ALTER TABLE "TicketHistoryRecord" ADD COLUMN "order" integer NULL;
+
+    UPDATE "Ticket" SET "order" = 100 WHERE "id" = '6ef3abc4-022f-481b-90fb-8430345ebfc2';
```

Comment:
It looks strange, what the ticket with this `id` is about?

Reply:
oops
It's a `open` status id.
Should be ` UPDATE "Ticket" SET "order" = 100 WHERE "status" = '6ef3abc4-022f-481b-90fb-8430345ebfc2';`

Reply:
I want to update the `order` here for all `tickets` that have the `open` status.

Reply:
Yes, it's correct.

Reply:
regenerate migration with correct update in b91bc7f

Reply:
Now it's correct.
Fix indentation of `UPDATE` statement, please, to be consistent.